packages/atomic/src/components/search/facets/atomic-automatic-facet-generator/atomic-automatic-facet-generator.tsx

```diff
@@ -70,7 +70,7 @@ export class AtomicAutomaticFacetGenerator implements InitializableComponent {
 
   private validateProps() {
     new Schema({
-      desiredCount: new NumberValue({min: 1, required: true}),
+      desiredCount: new NumberValue({min: 1, max: 10, required: true}),
```

Comment:
Is the max value coming from the API? If yes, wonder if it would be better documented if extracted out as a const with a note.

Reply:
Ideally, we could have a bunch of files with every default value in the search api as a const and use them in both Atomic and Headless. I guess it would be a whole new package ? it could even be something outside of ui-kit that is linked with the search api changes.

Reply:
I can add a some const in the files for now but it would be very helpful to have that with max, min and default values

Reply:
https://github.com/coveo/ui-kit/blob/7a5b21cf2af4db8864319bcc0184589c92d6f0d2/packages/headless/src/controllers/core/facets/_common/facet-option-definitions.ts#L62